SUI/SUI/MainViewModel.swift

```diff
@@ -0,0 +1,13 @@
+// MainViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Вью модель экрана c машинкой
+final class MainViewModel: ObservableObject {
+    // MARK: - Public properties
+
+    @Published var isCarClosed = false
+    @Published var isClimateShown = false
```

Comment:
```suggestion
    @Published var isClimateShown = false
```
    у тебя во вью есть свойство 
      @State var isClimateShown = false
      оно там не лишнее?